nucypher/cli/commands/deploy.py

```diff
@@ -364,14 +391,17 @@ def upgrade(general_config, actor_options, retarget, target_address, ignore_depl
             raise click.BadArgumentUsage(message="--target-address is required when using --retarget")
         if not actor_options.force:
             click.confirm(CONFIRM_RETARGET.format(contract_name=contract_name, target_address=target_address), abort=True)
-        receipt = ADMINISTRATOR.retarget_proxy(contract_name=contract_name,target_address=target_address)
+        receipt = ADMINISTRATOR.retarget_proxy(contract_name=contract_name,target_address=target_address, confirmations=0)
```

Comment:
Just checking - is `confirmations=0` intended here? Doesn't look like it was the default logic before.

Reply:
Good eye - that one is an over-zealous paste, meant to be `confirmations`.

Reply:
:heavy_check_mark: